wordpress-mobile/WordCamp-iOS
# Support Previews

Fixes #1 

Previews fail with a message similar to this:

```
'ContentView_Previews' is not a member type of 'WordCamp'

----------------------------------------

failedToBuildDylib: /Users/ctarda/Library/Developer/Xcode/DerivedData/WordCamp-bbnispueeptadngsfufaobibkesl/Build/Intermediates.noindex/Previews/WordCamp/Intermediates.noindex/WordCamp.build/Debug-iphonesimulator/WordCamp.build/Objects-normal/x86_64/ScheduleList.2.preview-thunk.swift:32:43: error: 'ContentView_Previews' is not a member type of 'WordCamp'
typealias ContentView_Previews = WordCamp.ContentView_Previews
                                 ~~~~~~~~ ^
```

## Changes
It seems like the source of the problem is that there is an entity in the codebase (`WordCamp`) with the same name as the application module. 

This PR changes:
- the name of the application module / binary / output.
- adds some previews to most views. For some views that would be a simple preview, for others the preview renders the view in both light and dark mode, for others, in two different devices

## Testing
- Checkout the branch, build and run, make sure the app still builds
- Check some of the previews
